### PR TITLE
Property reviews page

### PR DESCRIPTION
**Pull request by iamdobbs**
Added getPropertyById to property controller and updated route.
Created frontend propertyReviews and reviewsCard files. 
propertyReviews handles the GET to retrieve review info using property id.
reviewCard handles the functionality to render the review information for each individual review.
small update to loginForm to include resolution of the background image.

### CODE REVIEW

File: frontend/src/components/reviewCard/reviewCard.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { StarIcon } from "@heroicons/react/20/solid";
+import clsx from "clsx";
+import { format } from 'date-fns';
+
+//
```
Comment (joel-powell) [inline frontend/src/components/reviewCard/reviewCard.js]: empty comment

File: frontend/src/components/reviewCard/reviewCard.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { StarIcon } from "@heroicons/react/20/solid";
+import clsx from "clsx";
+import { format } from 'date-fns';
+
+// 
+
+function ReviewCard(props) {
```
Comment (joel-powell) [inline frontend/src/components/reviewCard/reviewCard.js]: can we de-structure review in the in the function params and change this to an arrow function:

`const ReviewCard = ({review}) => {`

File: frontend/src/components/reviewCard/reviewCard.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                )}
+                aria-hidden="true"
+              />
+            ))}
+          </div>
+          <p className="sr-only">{review.overallRating} out of 5 stars</p>
+
+          <div
+            className="prose prose-sm mt-4 max-w-none text-gray-500"
+            dangerouslySetInnerHTML={{ __html: review.comment }}
```
Comment (joel-powell) [inline frontend/src/components/reviewCard/reviewCard.js]: Don't wanna spoil the fun but can we please set the content of this div like we have the rest? 😅

File: frontend/src/components/reviewsList/reviewsList.js
Comment (joel-powell) [inline frontend/src/components/reviewsList/reviewsList.js:1]: Are we using this file?

**Review by joel-powell — CHANGES_REQUESTED** [review]

**Review by joel-powell — APPROVED** [review]

**Review by Ajmakovsky — APPROVED** [review]

**Pull request merged**